draft-ietf-jsonpath-base.md

```diff
@@ -2019,7 +2026,7 @@ Column "Change Controller" always has the value "IESG" and the column
 
 | Function Name | Brief description                  | Input                          | Output            |
 | length        | length of array                    | `OptionalValueType`                | `OptionalValueType`   |
-| count         | size of nodelist                   | `OptionalNodesType`                | `Value`           |
+| count         | size of nodelist                   | `NodelistType`                | `Value`           |
 | match         | regular expression full match      | `OptionalNodeOrValueType`, `Value` | `OptionalBooleanType` |
 | search        | regular expression substring match | `OptionalNodeOrValueType`, `Value` | `OptionalBooleanType` |
 {: #pre-reg title="Initial Entries in the Function Extensions Subregistry"}
```

Comment:
This table still refers to some of the types which have gone away.

Reply:
Yes.  Will update.